tests/e2e/monitoring/test_when_monitoring_nodes_are_shutdown.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.framework.testlib import tier4, E2ETest, bugzilla
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.monitoring import check_pvcdata_collected_on_prometheus
+from ocs_ci.utility import aws
+from ocs_ci.ocs.node import wait_for_nodes_status
+from tests.helpers import wait_for_resource_state
+from tests.sanity_helpers import Sanity
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def create_pods(pod_factory):
+    """
+    Create resource for the test
+    """
+
+    pod_objs = [
+        pod_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            status=constants.STATUS_RUNNING
+        ) for _ in range(2)
+    ]
+
+    # Check for the created pvc metrics on prometheus pod
+    for pod_obj in pod_objs:
+        assert check_pvcdata_collected_on_prometheus(pod_obj.pvc.name), (
+            f"On prometheus pod for created pvc {pod_obj.pvc.name} related data is not collected"
+        )
+    return pod_objs
+
+
+@bugzilla('1750328')
+@pytest.mark.polarion_id("OCS-711")
+class TestWhenShutdownAndRecoverOfPrometheusNode(E2ETest):
+    """
+    Validate whether shutdown and recovery of a node running
+    monitoring has no functional impact
+    """
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+        """
+        self.sanity_helpers = Sanity()
+
+    @tier4
+    def test_monitoring_shutdown_and_recovery_prometheus_node(self, create_pods, pod_factory):
+        """
+        Test case to validate whether shutdown and recovery of a
+        node where monitoring pods running has no functional impact
+        """
+        pod_objs = create_pods
+
+        aws_obj = aws.AWS()
```

Comment:
Please avoid using specific platform (aws in this case). 
Instead, use tests.conftest.nodes, as done in tests.manage.cluster.nodes.test_nodes_restart.TestNodesRestart.test_nodes_restart, so the test will run on VMWare too.